api/v1/helpers.go

```diff
@@ -421,6 +427,22 @@ func (s *Subcluster) GetStsSize(vdb *VerticaDB) int32 {
 	return s.Size
 }
 
+// GetVProxySize returns the number of replicas that will be assigned
+// to the vertica client proxy deployment. If sc.proxy.replica is not specified,
+// set it to half of the subcluster's size.
+func (s *Subcluster) GetVProxySize(vdb *VerticaDB) int32 {
+	if s.Proxy.Replica != 0 {
+		return s.Proxy.Replica
+	}
+
+	return s.Size % 2
```

Comment:
Sorry, let's stick with just replica for now.

Reply:
No problem. We can decide how to set the replica size later.